toplevel/vernac.ml

```diff
@@ -163,15 +171,16 @@ and load_vernac verbosely sid file =
      * raised, which means that we raised the end of the file being loaded *)
     while true do
       let loc, ast =
-        try Stm.parse_sentence !rsid in_pa
-        with
-        | Stm.End_of_input -> raise Stm.End_of_input
-        | any ->
-          let (e, info) = CErrors.push any in
-          let loc = Loc.get_loc info in
-          let msg = CErrors.iprint (e, info) in
-          Feedback.msg_error ?loc msg;
-          iraise (e, info)
+        (* try *)
+          Stm.parse_sentence !rsid in_pa
+        (* with *)
+        (* | any when not is_end_of_input any -> *)
+        (*   let (e, info) = CErrors.push any in *)
+        (* Re-enable when we switch back to feedback-based error printing *)
```

Comment:
This seems to be placed in the wrong position. IMHO you should write a function `protect f x = try f x with ...` and put the comment there